testgrid/config.yaml

```
@@ -7804,6 +7813,15 @@ dashboards:
   - name: pr-k8s-e2e
     description: Run e2e tests
     test_group_name: pull-kubernetes-e2e-aks-engine-azure
+  - name: pr-cloud-provider-e2e
+    description: Run Kubernetes e2e tests for cloud-provider-azure
+    test_group_name: pull-cloud-provider-azure-e2e
+  - name: pr-cloud-provider-unit
+    description: Run unit tests for cloud-provider-azure
+    test_group_name: pull-cloud-provider-azure-unit
+  - name: pull-cloud-provider-check
```

Nit: seems like you name everything else in this dash with `pr-`
```suggestion
  - name: pr-cloud-provider-check
```

---

thanks, good catch